### PR TITLE
Feat/ratings

### PR DESCRIPTION
**Pull request by soyamore**
- Enable the indexation of review ratings

### CODE REVIEW

**Review by Fifciu — CHANGES_REQUESTED** [review]
Good catch. Please add info to the Changelog and I can merge it then

**Review by gibkigonzo — APPROVED** [review]

**Comment by soyamore** [comment]
@Fifciu Hello Filip change log is added to the PR

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [1.11.12] - 2020-11-12
```
Comment (Fifciu) [inline CHANGELOG.md:6]: It is 1.11.2

**Review by Fifciu — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [1.11.2] - 2020-11-12
```
Comment (soyamore) [inline CHANGELOG.md:6]: @Fifciu it's good now,

**Review by Fifciu — APPROVED** [review]

**Comment by Fifciu** [comment]
Great, thanks!

**Pull request merged**